### PR TITLE
update Dockerfile for hub

### PR DESCRIPTION
**Pull request by General-Beck**
update to Ubuntu Xenial 16.04

### CODE REVIEW

File: docker/hub/Dockerfile [hunk 1]
```diff
@@ -52,6 +51,7 @@ cd /build&&git clone https://github.com/paritytech/parity && \
         # add-apt-repository
         software-properties-common \
         make \
+	cmake \
```
Comment (eosclassicteam) [inline docker/hub/Dockerfile]: Please fix typo here

**Comment by ddorgan** [comment]
[clabot:check]

**Comment by ddorgan** [comment]
[clabot:check]

**Comment by andresilva** [comment]
The build for this PR has run and passed (https://gitlab.parity.io/parity/parity/-/jobs/92370). The PR status is not being updated and I can't force the merge.

**Pull request closed without merging**